azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/Account.java

```diff
@@ -132,7 +128,7 @@ public void selectSubscription(List<String> selectedSubscriptionIds) {
     protected abstract Mono<TokenCredentialManager> createTokenCredentialManager();
 
     public Mono<Boolean> checkAvailable() {
```

Comment:
Shall we add java doc here to indicate this method will throw auth exception if not available?